.ci_support/linux_64_python3.6.____cpython.yaml

```diff
@@ -33,7 +33,7 @@ python:
 target_platform:
 - linux-64
 volk:
-- '2.4'
+- '2.4.1'
```

Comment:
It isn't necessary to add the ".1" here. It will build against the latest version of 2.4. But for now it doesn't hurt.

The way these .ci_support files work is that they are generated by `conda-smithy` whenever the recipe is re-rendered. The particular values for these pins come from the global pinning file installed by `conda-forge-pinning`. They get updated with PRs to that feedstock, and then there are bots that handle PRs to re-render any affected recipes. But since we're not ready to globally update to gnuradio_core 3.9 yet, updating these manually is perfectly fine. Just know that they will get overwritten whenever the recipe is re-rendered.

Reply:
Thanks for the explanation. I have zero experience about how these things work. I guess we can leave it as '2.4.1' if it's going to be overwritten evntually.